src/include/splash/DataCollector.hpp

```diff
@@ -52,10 +52,13 @@
 #include "splash/Dimensions.hpp"
 #include "splash/Selection.hpp"
 #include "splash/core/DCDataSet.hpp"
+#include "splash/core/splashMacros.hpp"
 
 namespace splash
 {
 
+    class AttributeInfo;
```

Comment:
it might be useful to create an `AttributeInfo.def` file and include it in `AttributeInfo.hpp` (and here) to expose this dependency more openly.